test-adapter/src/main/scala/org/scalajs/sbttestadapter/ScalaJSRunner.scala

```diff
@@ -61,28 +63,37 @@ final class ScalaJSRunner private[testadapter] (
     taskInfos.map(ScalaJSTask.fromInfo(this, _)).toArray
   }
 
-  def done(): String = synchronized {
-    ensureNotDone()
-
-    stopSlaves()
-
-    master.send("runnerDone")
-    val summary = ComUtils.receiveResponse(master) {
-      case ("ok", summary) => summary
+  def done(): String = {
+    synchronized {
+      ensureNotDone()
     }
```

Comment:
I was afraid of that. But wasn't there already another place where we assumed `done()` would be called by sbt exactly once? Under that assumption (which may or may not be valid), it's ok to split it, isn't it? So if we assume it somewhere else, what's the point of being resilient here? (Of course if we don't assume that anywhere else, I'll find another way).